src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -833,46 +826,56 @@ class AbstractUtxoCoin extends BaseCoin {
    */
```

Comment:
I can't leave a comment on line 825, but the @ returns should be updated.

Reply:
Done

Reply:
actually we should add the typescript signature instead in the future